docs/reference/server-config.md

```diff
@@ -29,29 +29,30 @@ In this section, you'll learn how to configure the K3s server.
-| `--etcd-s3-skip-ssl-verify`           | N/A                      | Disables S3 SSL certificate validation                                                                   |
-| `--etcd-s3-access-key` value          | `AWS_ACCESS_KEY_ID`      | S3 access key                                                                                            |
-| `--etcd-s3-secret-key` value          | `AWS_SECRET_ACCESS_KEY`  | S3 secret key                                                                                            |
-| `--etcd-s3-bucket` value              | N/A                      | S3 bucket name                                                                                           |
-| `--etcd-s3-region` value              | N/A                      | S3 region / bucket location (optional) (default: "us-east-1")                                            |
-| `--etcd-s3-folder` value              | N/A                      | S3 folder                                                                                                |
-| `--etcd-s3-insecure`           | Disables S3 over HTTPS |
-| `--etcd-s3-timeout` value     | S3 timeout (default: 30s) |
+| Flag                                  | Environment Variable       | Description                                                                                              |
+| ------------------------------------- | -------------------------- | -------------------------------------------------------------------------------------------------------- |
+| `--datastore-endpoint` value          | `K3S_DATASTORE_ENDPOINT`   | Specify etcd, Mysql, Postgres, or Sqlite (default) data source name                                      |
+| `--datastore-cafile` value            | `K3S_DATASTORE_CAFILE`     | TLS Certificate Authority file used to secure datastore backend communication                            |
+| `--datastore-certfile` value          | `K3S_DATASTORE_CERTFILE`   | TLS certification file used to secure datastore backend communication                                    |
+| `--datastore-keyfile` value           | `K3S_DATASTORE_KEYFILE`    | TLS key file used to secure datastore backend communication                                              |
+| `--etcd-expose-metrics`               | N/A                        | Expose etcd metrics to client interface (default: false)                                                 |
+| `--etcd-disable-snapshots`            | N/A                        | Disable automatic etcd snapshots                                                                         |
+| `--etcd-snapshot-name` value          | N/A                        | Set the base name of etcd snapshots. Default: etcd-snapshot-`<unix-timestamp>` (default:"etcd-snapshot") |
+| `--etcd-snapshot-schedule-cron` value | N/A                        | Snapshot interval time in cron spec. eg. every 5 hours '\* _/5 _ \* _' (default: "0 _/12 \* \* \*")      |
```

Comment:
I thought there was a PR out there to change this flag, as the current example expression takes a snapshot every minute of every 5th hour. I'm not finding it now though.

Reply:
Your right, I'm not sure if that got merged yet, but I will change the value to match it.